app/models/user.rb

```diff
@@ -27,6 +27,7 @@ class User < ApplicationRecord
   #associations
   belongs_to :organisation
   belongs_to :role
+  has_one_attached:image
```

Comment:
@apeksha2697  Please fix indentation, there should be space btw has_one_attached : image

Reply:
done